core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestMultipleDistinctAggregationToMarkDistinct.java

```diff
@@ -171,14 +173,15 @@ public void testGlobalAggregation()
     @Test
     public void testAggregationNDV()
     {
-        PlanNodeId aggregationNodeId = new PlanNodeId("aggregationNodeId");
+        PlanNodeId aggregationId = new PlanNodeId("aggregationId");
```

Comment:
There should be no changes to existing plan tests in this commit right ?
Are these just just cosmetic changes ? If so, they shouldn't be in this commit